packages/dm-core-plugins/src/form/templates/ObjectModelUncontainedTemplate.tsx

```diff
@@ -27,53 +27,85 @@ export const ObjectModelUncontainedTemplate = (
       ? uiAttribute?.showExpanded
       : config.showExpanded
   )
+
   const value = watch(namePath)
   const { dataSource, documentPath } = splitAddress(idReference)
   const address =
     value && value.address && value.referenceType === 'link'
       ? resolveRelativeAddress(value.address, documentPath, dataSource)
       : undefined
+
+  const isExpandable = useMemo(
+    () => address && !(onOpen && !uiAttribute?.showInline),
+    [address]
+  )
   return (
-    <Fieldset>
-      <Legend>
-        <Typography bold={true}>{getDisplayLabel(attribute)}</Typography>
-        {!config.readOnly && (
-          <SelectReference
-            attributeType={attribute.attributeType}
-            namePath={namePath}
-          />
-        )}
-        {attribute.optional && address && !config.readOnly && (
-          <RemoveObject namePath={namePath} />
-        )}
-        {address && !(onOpen && !uiAttribute?.showInline) && (
-          <TooltipButton
-            title={isExpanded ? 'Collapse' : 'Expand'}
-            button-variant='ghost_icon'
-            button-onClick={() => setIsExpanded(!isExpanded)}
-            icon={isExpanded ? chevron_up : chevron_down}
-          />
-        )}
-        {address && onOpen && !uiAttribute?.showInline && (
-          <OpenObjectButton
-            viewId={namePath}
-            viewConfig={{
-              type: 'ReferenceViewConfig',
-              scope: '',
-              recipe: uiRecipe?.name,
+    <div>
+      <legend className='flex flex-row h-10 justify-between border-b-[2px]'>
```

Comment:
`flex-row` is redundant, and the `border-b-2` is already 2px
```suggestion
      <legend className='flex h-10 justify-between border-b-2'>
```